Assets/MRTK.ThirdParty/MRTK-Quest/Scripts/Input/DeviceManager/Controllers/OculusQuestHand.cs

```diff
@@ -164,9 +171,14 @@ protected bool IsInTeleportPose
                 if (MRTKOculusConfig.Instance.ActiveTeleportPointerMode == MRTKOculusConfig.TeleportPointerMode.None) return false;
                 if (!TryGetJoint(TrackedHandJoint.Palm, out var palmPose)) return false;
 
-                Transform cameraTransform = CameraCache.Main.transform;
+                Camera mainCamera = CameraCache.Main;
+
+                if (mainCamera == null)
+                {
+                    return false;
+                }
 
-                Vector3 projectedPalmUp = Vector3.ProjectOnPlane(-palmPose.Up, cameraTransform.forward);
```

Comment:
Please restore this line of code, and I'll approve this change!

Reply:
Actually, it was intentional - variable projectedPalmUp is not used in IsInTeleportPose (probably a copy-paste mistake).

Reply:
Ah you're right, it might be a leftover piece from when I was working on the teleport pose.